Example/NWSTokenView.xcodeproj/xcuserdata/James.xcuserdatad/xcschemes/NWSTokenView-Example (Release).xcscheme

```diff
@@ -1,112 +0,0 @@
-<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
@wfilleman Why were the schemes updated? This was necessary to install on a device without provisioning.